kansha/board/save_template.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding:utf-8 -*-
+#--
+# Copyright (c) 2012-2014 Net-ng.
+# All rights reserved.
+#
+# This software is licensed under the BSD License, as described in
+# the file LICENSE.txt, which you should have received as part of
+# this distribution.
+#--
+
+from nagare import component, editor, presentation, validator as nagare_validator
+from nagare.i18n import _
+
+from kansha import validator
+from kansha.toolbox import remote
+
+
+class SaveTemplateEditor(editor.Editor):
```

Comment:
Don't inherit from `editor.Editor` if you don't use its core API (change `if self.is_validated(('title', 'shared'))` by `if title.error is None and shared.error is None`